XCUITests/DataManagementTests.swift

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+
+class DataManagementTests: BaseTestCase {
+
+    //Testing the entries are correctly added and deleted. This a termporary behaviour. This test will be changed after releasing Firefox 15.x and Bug 1499074 will be fixed.
+    func testWebSiteDataEnterFirstTime() {
+        navigator.goto(WebsiteDataSettings)
+        let expectedWebsiteDataEntries = app.tables.cells.count
+        XCTAssertEqual(expectedWebsiteDataEntries, 2)
+        navigator.performAction(Action.AcceptClearAllWebsiteData)
+        let expectedWebsiteDataEntries2 = app.tables.cells.count
+        XCTAssertEqual(expectedWebsiteDataEntries2, 1)
+        navigator.createNewTab()
+        navigator.goto(WebsiteDataSettings)
+        let expectedWebsiteDataEntries3 = app.tables.cells.count
+        XCTAssertEqual(expectedWebsiteDataEntries3, 2)
+    }
+    
+    // Testing the search bar, search results and 'Show More' option.
+    func testWebSiteDataOptions() {
+        // Visiting some websites to create Website Data needed to reveal the "Show More" button
+        let WebsiteData = ["www.facebook.com", "www.youtube.com", "www.twitter.com", "www.google.com", "www.facebook.com", "www.mozilla.org"]
```

Comment:
Naming: variable name should start with lower case letter. Also, I would pick a more descriptive name, this is too general.
For example, websitesList or websitesToBeOpen...